app/src/sharedTest/java/org/oppia/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -68,6 +71,13 @@ class ProfileChooserFragmentTest {
       .build()
       .inject(this)
   }
+  @Test
+  fun testProfileChooserFragment_configurationChange_checkSpanCount_spanCountVerifiedSucessfully() {
```

Comment:
Typo in last word, it should be `Successfully`

Reply:
Done